README.md

```diff
@@ -92,6 +92,14 @@ Contains AWS Cli + AWS EB Cli + CI Helper + jq
 
 Amazon Linux based image containing Systemd for service management in docker container.
 
+### Azure
+
+Contains Azure Cli + Terraform.
+
+### Chrome
+
+Contains Chromium browser (installed with Node:10-alpine).
```

Comment:
```suggestion
Contains Chromium browser.
```
to make future updates easier

Reply:
Or we could juste write this ?
```suggestion
Contains Chromium browser and the latest Node LTS.
```
That way the user knows Node is available and we don't have to edit it either.